### ISSUE TITLE
Directory name is incorrect: 'C:\Users\x07dministrator'

### ISSUE THREAD

**Issue by mihaiplesa** [brave/brave-browser#6593]
```
19:35:25  WARNING: Your metrics.cfg file was invalid or nonexistent. A new one will be created
19:35:25  .
19:35:25  
19:35:25  
19:35:25  src (ERROR)
19:35:25  ----------------------------------------
19:35:25  [0:00:00] Started.
19:35:25  ---------------------------------------
19:35:25  -
19:35:25  Traceback (most recent call last):
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\metrics.py", line 267, in print_notice_and_exit
19:35:25  
19:35:25      yield
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient.py", line 3158, in <module>
19:35:25  
19:35:25      sys.exit(main(sys.argv[1:]))
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient.py", line 3144, in main
19:35:25  
19:35:25  
19:35:25      return dispatcher.execute(OptionParser(), argv)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\subcommand.py", line 252, in execute
19:35:25  
19:35:25      return command(parser, args[1:])
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient.py", line 2701, in CMDsync
19:35:25  
19:35:25      ret = client.RunOnDeps('update', args)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient.py", line 1739, in RunOnDeps
19:35:25  
19:35:25      patch_refs=patch_refs, target_branches=target_branches)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient_utils.py", line 917, in flush
19:35:25  
19:35:25      reraise(e[0], e[1], e[2])
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient_utils.py", line 994, in run
19:35:25  
19:35:25      self.item.run(*self.args, **self.kwargs)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient.py", line 916, in run
19:35:25  
19:35:25      file_list)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient_scm.py", line 132, in RunCommand
19:35:25  
19:35:25      return getattr(self, command)(options, args, file_list)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient_scm.py", line 545, in update
19:35:25  
19:35:25      self._UpdateMirrorIfNotContains(mirror, options, rev_type, revision)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient_scm.py", line 988, in _UpdateMirrorIfNotContains
19:35:25  
19:35:25      lock_timeout=getattr(options, 'lock_timeout', 0))
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\git_cache.py", line 592, in populate
19:35:25  
19:35:25      gclient_utils.safe_makedirs(self.GetCachePath())
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\gclient_utils.py", line 303, in safe_makedirs
19:35:25  
19:35:25      os.makedirs(tree)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\bootstrap-3_8_0b1_chromium_1_bin\python\bin\lib\os.py", line 150, in makedirs
19:35:25  
19:35:25      makedirs(head, mode)
19:35:25  
19:35:25    File "C:\kies\vendor\depot_tools\bootstrap-3_8_0b1_chromium_1_bin\python\bin\lib\os.py", line 157, in makedirs
19:35:25  
19:35:25      mkdir(name, mode)
19:35:25  
19:35:25  WindowsError: [Error 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Users\x07dministrator'
```

**Comment by bsclifton** [comment]
Would be good to:
- reproduce the problem
- see if there is a verbose log level we can get from gclient
- understand why the second `\` isn't being double escaped (likely, a variable is referencing `$HOME`, which equates to `%USERPROFILE% on Windows)

It could be a string replace not doing global, if it's a regex replace. ex:
`/\\/\\\\`
vs
`/\\/\\\\/g`

**Comment by bsclifton** [comment]
A recent example: https://ci.brave.com/job/brave-browser-build-pr/job/pr3785_maxk-fix-clear-cookies_0.72.x/1/execution/node/355/log/

**Comment by bsclifton** [comment]
Possibly fixed with https://github.com/brave/brave-browser/pull/6672

**Comment by bsclifton** [comment]
Possible fix merged back to 0.71.x; seems to be working so far! Thanks, @iefremov 😄
